flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/wrappers/AbstractStormSpoutWrapper.java

```diff
@@ -97,9 +106,42 @@ public AbstractStormSpoutWrapper(final IRichSpout spout,
 	}
 
 	@Override
+	public void open(Configuration parameters) throws Exception {
+		stormConf = new HashMap();
+
+		/* parameters is task configuration, we can get storm configuration only from job configuration */
+		RuntimeContext ctx = super.getRuntimeContext();
+		if (ctx instanceof StreamingRuntimeContext) {
+			Configuration jobConfiguration = ((StreamingRuntimeContext) ctx).getJobConfiguration();
+
```

Comment:
The open() is usually called by openAllOperators(), and the Configuration config parameter is usually task Configuration, not job Configuration.